vite-umd.config.js

```diff
@@ -0,0 +1,56 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import fs from 'fs/promises';
+
+/**
+* Vite configuration
+*/
+export default defineConfig({
+  build: {
+    emptyOutDir: false,
+    lib: {
+      entry: './src/index.js',
+      fileName: (format) => (format === 'umd' ? 'mirador.js' : undefined),
```

Comment:
Question/suggestion about clarifying the naming of these files and extensions. Are we mixing which version we consider "default"? `package.json` (with `type="module"`) and the filename `vite.config.js` imply that es is the default. But in `/dist`, `mirador.js` is actually the UMD build.

The Vite docs show this pattern:
```
$ vite build
building for production...
dist/my-lib.js      0.08 kB / gzip: 0.07 kB
dist/my-lib.umd.cjs 0.30 kB / gzip: 0.16 kB
Recommended package.json for your lib:

package.json (single entry)
{
  "name": "my-lib",
  "type": "module",
  "files": ["dist"],
  "main": "./dist/my-lib.umd.cjs",
  "module": "./dist/my-lib.js",
  "exports": {
    ".": {
      "import": "./dist/my-lib.js",
      "require": "./dist/my-lib.umd.cjs"
    }
  }
}
```

Reply:
I think we're just following the pattern from Mirador 3 (and it seems like an ok one... ES build users don't care what the artifact is called, but UMD users probably do?)